Makefile

```diff
@@ -38,6 +41,7 @@ BUILD_FLAGS ?=
 BUILD_TAGS ?=
 BUILD_CGO_ENABLED := 0
 PD_EDITION ?= Community
+GO       := GO111MODULE=on go
```

Comment:
I think we could replace all `GO111MODULE=on go` with this variable?

Reply:
yes ,it will move.